__tests__/costumes.test.js

```diff
@@ -153,7 +73,7 @@ describe("createCostume adapter", () => {
             SELECT * FROM costumes WHERE name='buttless chaps';
         `);
         
-        expect(matchesCostumeInDatabase(buttlessChaps, chapsFromDatabase)).toBe(true);
+        expect(matchesDatabase(buttlessChaps, chapsFromDatabase)).toBe(true);
```

Comment:
prefer to manually check the expected properties
```suggestion
        expect(buttlessChaps.name).toBe("...");
        expect(buttleChaps.price).toBe(XYZ);
```

Reply:
Is that because this actually tests the value of each property instead of just matching the object properties?

Reply:
Let me put it this way:

`expect(buttlessChaps.name).toEqual("Some test name")` is a perfectly adequate test case, and will test both that the key is correct/not undefined AND the value is as expected

Reply:
Thank you for explaining that. Makes total sense!